TESTS/mbed_drivers/flashiap/main.cpp

```diff
@@ -220,6 +222,9 @@ void flashiap_timing_test()
     utest_printf("\nFlash timing:\n");
     uint32_t sector_size = flash_device.get_sector_size(end_address - 1UL);
     uint32_t base_address = end_address - sector_size;
+    utest_printf("ROM ends at 0x%lx, test starts at 0x%lx\n", FLASHIAP_APP_ROM_END_ADDR, base_address);
+    TEST_SKIP_UNLESS_MESSAGE(base_address >= FLASHIAP_APP_ROM_END_ADDR, "Test skipped. Test region overlaps code.");
```

Comment:
You are also adding some debug prints , not only skip test?

Reply:
I can remove them, but it's practical in case of test skip to understand.

Reply:
I was referring to the commit message and PR title (does not cover adding debug msg). Otherwise looks fine to me